Test/Utils/ContrailAPI_New/Models/FloatingIpPool.ps1

```diff
@@ -0,0 +1,25 @@
+class FloatingIpPool : BaseResourceModel {
+    [String] $Name
+    [String] $NetworkName
```

Comment:
Why isn't it NetworkFqdn? In FloatingIP there is PoolFqName, why is it different?

Reply:
Good point. Fixed. I need to work on my consistency in things like this. Thanks! :)